[2024-05-26T13:51:53.122+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_very_first_dag.my_second_task manual__2024-05-26T13:51:44.404443+00:00 [queued]>
[2024-05-26T13:51:53.156+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_very_first_dag.my_second_task manual__2024-05-26T13:51:44.404443+00:00 [queued]>
[2024-05-26T13:51:53.156+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-05-26T13:51:53.211+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): my_second_task> on 2024-05-26 13:51:44.404443+00:00
[2024-05-26T13:51:53.224+0000] {standard_task_runner.py:60} INFO - Started process 5529 to run task
[2024-05-26T13:51:53.239+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'my_very_first_dag', 'my_second_task', 'manual__2024-05-26T13:51:44.404443+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpstrmkiy5']
[2024-05-26T13:51:53.250+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask my_second_task
[2024-05-26T13:51:53.296+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-05-26T13:51:53.373+0000] {task_command.py:423} INFO - Running <TaskInstance: my_very_first_dag.my_second_task manual__2024-05-26T13:51:44.404443+00:00 [running]> on host ac6c883b25f7
[2024-05-26T13:51:53.619+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_very_first_dag' AIRFLOW_CTX_TASK_ID='my_second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-26T13:51:44.404443+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-26T13:51:44.404443+00:00'
[2024-05-26T13:51:53.621+0000] {logging_mixin.py:188} INFO - Hello from Airflow again
[2024-05-26T13:51:53.622+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-26T13:51:53.645+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=my_very_first_dag, task_id=my_second_task, execution_date=20240526T135144, start_date=20240526T135153, end_date=20240526T135153
[2024-05-26T13:51:53.749+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-26T13:51:53.794+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
